dune-project

```
@@ -1,2 +1,77 @@
 (lang dune 2.3)
+
 (name unison)
+
+(version dev)
+
+(generate_opam_files false)
+
+(license GPL-3.0-or-later)
+
+(maintainers "maintainers")
+
+(authors "Trevor Jim, Benjamin C. Pierce and J\195\169r\195\180me Vouillon")
+
+(source
+ (github bcpierce00/unison))
+
+(homepage https://www.cis.upenn.edu/~bcpierce/unison/)
+
+(documentation "https://github.com/bcpierce00/unison/wiki")
+
+(package
+ (name unison)
+ (synopsis "File-synchronization tool for Unix and Windows")
+ (description "
+Text based userinterface for Unison.
```

Avoid empty line in description.
```suggestion
 (description "Text based userinterface for Unison.
```